samples/peripheral/802154_phy_test/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  name: IEEE 802.15.4 PHY Test Tool
+  description: IEEE 802.15.4 PHY Test Tool sample
+tests:
+  samples.peripheral.802154_phy_test:
+    build_only: true
+    build_on_all: true
+    platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpunet
```

Comment:
This needs `integration_platforms:` section - see other samples. And as per https://github.com/nrfconnect/sdk-nrf/pull/5250, `build_on_all:` should no longer be used.

Reply:
Added integration_platforms, removed build_on_all